src/tokens/rsk.ts

```diff
@@ -0,0 +1,29 @@
+import { ChainId } from '../chainIds';
+import { IToken } from '../interfaces';
+
+export const tokens: IToken[] = [
+  {
+    name: 'RBTC',
+    address: '0x0000000000000000000000000000000000000000',
+    chainId: ChainId.RSKTestnet,
```

Comment:
ChainId is RSK, same for the other tokens

Reply:
Done!